src/components/Drawer/types.ts

```diff
@@ -1,10 +1,16 @@
 import { AcceptedIconNames } from 'components/Icon/types';
+import * as H from 'history';
+import { match } from 'react-router';
 
 export type MenuItem = {
   name: string;
   url: string;
   state?: Record<string, any> | null;
   visible: boolean;
   iconName: AcceptedIconNames;
+  isActive?<Params extends { [K in keyof Params]?: string }>(
```

Comment:
maybe you should also expose a generic type for LocationState and partially import the Location and LocationState types. This will help us because we have an optional state prop in the MenuItem.

example
```
import { Location, LocationState } from 'history';


isActive?:<Params extends { [K in keyof Params]?: string }, S = LocationState>(
    match: match<Params> | null,
    location: Location<S>
  ): boolean;
```